core/src/main/java/io/questdb/cutlass/text/ParallelCsvFileImporter.java

```diff
@@ -340,7 +340,7 @@ public void of(
         this.tableName = tableName;
```

Comment:
We need to pass security context to this method and use it when creating a new table. Currently this class bypasses security by making a direct call to `TableUtils#createTable()`.

Reply:
this is a good point, we might need to check createTable perm, lets make it WIP

Reply:
Marking this one as resolved as it's something to be addressed in a future PR.